marketplace-client/src/marketplace/src/app/account/register-stuff/register-stuff.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="card m-3">
+  <h1 class="text-center card-header">Register stuff member</h1>
+  <div *ngIf="!registered" class="card-body">
+    <form [formGroup]="form" (ngSubmit)="onSubmit()">
+      <div class="form-row">
+        <div class="form-group col-6">
+          <label>First Name*</label>
+          <input type="text" formControlName="name" class="form-control" [ngClass]="{ 'is-invalid': submitted && getForm.name.errors }" />
```

Comment:
it's much more readable to have 1 line = 1 attr

Reply:
Yep, it is indeed better to do it the way you described. This will be fixed.